src/main/java/org/checkerframework/checker/https/HTTPSAnnotatedTypeFactory.java

```diff
@@ -46,4 +48,16 @@ private void setStringToHTTPS(Tree tree, AnnotatedTypeMirror type){
             }
         }
     }
+
+    private void checkConcatenatedString (Tree tree, AnnotatedTypeMirror type) {
+        if (tree.getKind().equals(Tree.Kind.PLUS)){
+            BinaryTree binaryTree = (BinaryTree) tree;
+            if(AnnotationUtils.containsSameByClass(getAnnotatedType(binaryTree.getLeftOperand()).getAnnotations(),
```

Comment:
This line is really long. I'd use a local variable (of type `AnnotatedTypeMirror`) for `getAnnotatedType (binaryTree.getLeftOperand())`. That will make this more readable.